source/http-boot/pmem_node.md

```diff
@@ -0,0 +1,131 @@
+<!--SPDX-License-Identifier: CC-BY-SA-4.0-->
+
+# About This Document
+
+This document describes a proposal for providing the operating system with access
+to the boot image, exposed as a ramdisk via a PMEM node on Device Tree–based
+platforms.
+
+In this document, the term *firmware* is used generically to refer to platform
+firmware components involved in the boot process, including the bootloader
+(e.g., U-Boot), where applicable.
+
+## Overview
+
+When the kernel is booted from a Linux live image or an installer image
+(where installation requires access to local packages),
+the boot image must remain accessible after the kernel starts.
```

Comment:
We can make this a more verbose. e.g 
The local packages must remain accessible after the kernel calls ExitBootServices. Invoking ExitBootServices will make all the local mounted drives dissapear